src/adapters/git.ts

```diff
@@ -33,7 +35,9 @@ abstract class GitAdapter implements IRepoAdapter {
       // Repo already exists; just fetch
       await this.git(repo).fetch('origin');
     } else {
-      await simpleGit().clone(repoPath, localPath);
+      const git = simpleGit();
+      git.silent(true);
```

Comment:
Why did we remove logging here?

Reply:
No logging is actually the default, see the `git()` method in this class. We disable logging because we'll print errors if they occur, they'll be duplicated if we also allow the git package to log them. We can't use `this.git(...)` here because the directory doesn't exist yet. This just makes it consistent with other usages.